curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/initializing-variables-with-the-assignment-operator.md

```diff
@@ -23,7 +23,7 @@ Define a variable `a` with `var` and initialize it to a value of `9`.
 You should initialize `a` to a value of `9`.
 
 ```js
-assert(/var\s+a\s*=\s*9\s*/.test(code));
+assert(/var\s+a\s*=\s*\b9\b\s*/.test(code));
```

Comment:
```suggestion
assert(/var\s+a\s*=\s*9(\s*;?\s*)$/.test(code));
```

Reply:
Thanks for the suggestion @Sky020 . Will keep the triage label in mind next time. Also yea will make sure to create new branch for the fixes next time.